### PR TITLE
[ContainerInstance]Generated the track 2 SDK for ContainerInstance.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description
Generated code and wrote test cases.
## Work Content details
+ Generated track 2 SDK code.
+ Wrote test cases and successfully record.
+ Modified assembly version(1.0.0-beta.1) and namespace(ContainerInstance to Microsoft.ContainerInstance ).

### CODE REVIEW

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-preview.1 (Unreleased)
```
Comment (HarveyLink) [inline sdk/containerinstance/Azure.ResourceManager.ContainerInstance/CHANGELOG.md:3]: Update the version and release date.

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+
+## Getting started 
+
+### Install the package
+
+Install the Azure ContainerInstance management library for .NET with [NuGet](https://www.nuget.org/):
+
+```PowerShell
+Install-Package Azure.ResourceManager.ContainerInstance -Version 1.0.0-preview.1
```
Comment (HarveyLink) [inline sdk/containerinstance/Azure.ResourceManager.ContainerInstance/README.md:12]: Update version

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/CHANGELOG.md (hunk 1 above)
Comment (LucasYao93) [reply sdk/containerinstance/Azure.ResourceManager.ContainerInstance/CHANGELOG.md]: Done, But uncertain release date.

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/README.md (hunk 2 above)
Comment (LucasYao93) [reply sdk/containerinstance/Azure.ResourceManager.ContainerInstance/README.md]: Done

**Comment by isra-fel** [comment]
Should the pipeline config be updated to pick up this new SDK project in CI and how?

**Comment by HarveyLink** [comment]
> Should the pipeline config be updated to pick up this new SDK project in CI and how?

The previous Pipeline CI PR should be merged before this one: #18349

**Comment by LucasYao93** [comment]
HI @allenjzhang,
For fix swagger issue, I used the specified modelerfour version(4.15.448) which cause CI(Build Analyze) to fail.

Thanks,
Lucas

**Comment by LucasYao93** [comment]
> HI @allenjzhang,
> For fix swagger issue, I used the specified autorest/modelerfour version(4.15.448) which cause CI(Build Analyze) to fail.
> 
> Thanks,
> Lucas

**Comment by isra-fel** [comment]
Issue was fixed after syncing master. @YalinLi0312 could you take a look again? Thank you.

**Review by allenjzhang — APPROVED** [review]

**Comment by isra-fel** [comment]
Thanks Allen for approving.
As we disscuessed in mails, we are going to put this pull request on hold.

**Pull request closed without merging**